internal/vercheck/vercheck.go

```diff
@@ -15,32 +15,60 @@ import (
 
 	"github.com/fatih/color"
 	"github.com/pkg/errors"
-	"golang.org/x/mod/semver"
-
+	"github.com/samber/lo"
 	"go.jetpack.io/devbox/internal/boxcli/usererr"
 	"go.jetpack.io/devbox/internal/build"
 	"go.jetpack.io/devbox/internal/cmdutil"
 	"go.jetpack.io/devbox/internal/envir"
 	"go.jetpack.io/devbox/internal/ux"
 	"go.jetpack.io/devbox/internal/xdg"
+	"golang.org/x/mod/semver"
 )
 
 // Keep this in-sync with latest version in launch.sh.
 // If this version is newer than the version in launch.sh, we'll print a notice.
 const expectedLauncherVersion = "v0.2.0"
 
+// versionCheckEnvName determines whether the version check has already occurred.
+// We set this env-var so that this devbox command invoking other devbox commands
+// do not re-run the version check and print the notice again.
+const versionCheckEnvName = "DEVBOX_VERSION_CHECK"
```

Comment:
do we want __DEVBOX_VERSION_CHECK?

Reply:
I've been inconsistent, but I think starting with `__` is better.

I try to follow:

`DEVBOX_[name]` when it is a variable that a user might want to set/change.

`__DEVBOX_[name]` for automated values that the user should never touch. (i.e. private)